SYCL/Matrix/element_wise_irreg_sum_rows.cpp

```diff
@@ -89,14 +89,14 @@ void matrix_sum_rows(queue q, big_matrix<T, M, N> &B, nd_range<2> &r) {
              for (int i = 0; i < data.length() / (TK / 4); i++) { // 4 per row
                // i*SG_SIZE index is found based on the round robin
                // distribution we are using in the implementation
-               sum_local_rows[row + global_idx * (TK / 4)] += data[i + row * 4]
+               sum_local_rows[row + global_idx * (TK / 4)] += data[i + row * 4];
```

Comment:
@myler, this code was missing the semi colon but we were not getting the analysis that this test is failing at compile time.
How can we improve sharing the analysis of failures on the matrix code?